vaadin-side-nav-flow-parent/vaadin-side-nav-flow/src/test/java/com/vaadin/flow/component/sidenav/tests/SideNavItemTest.java

```diff
@@ -388,6 +391,106 @@ public void expandAndCollapse_isCollapsed() {
         Assert.assertFalse(sideNavItem.isExpanded());
     }
 
+    @Test
+    public void createWithNoPathAlias_pathAliasesEmpty() {
+        final SideNavItem item = new SideNavItem("Test");
+
+        Assert.assertEquals(0, item.getPathAliases().size());
```

Comment:
Let's also assert that the `pathAliases` property is set correctly in this and following tests.

Reply:
Added the assertions.